Composer/packages/lib/shared/src/skillsUtils/index.ts

```diff
@@ -101,5 +101,48 @@ export const isSkillHostUpdateRequired = (skillHostEndpoint?: string) => {
   return !skillHostEndpoint || isLocalhostUrl(skillHostEndpoint);
 };
 
+/**
+ * Returns true if the runtime key is adaptive; false otherwise.
+ * Example adaptive runtime keys: 'adaptive-runtime-dotnet-webapp' or 'adaptive-runtime-node-functions'
+ * Example older adaptive runtime keys: 'csharp-azurewebapp-v2'
```

Comment:
FWIW, this was used for a pretty brief period of time during the early stages of the runtimes. Only people running preview bits would have  projects with the `csharp-azurewebapp-v2` value.